frontend/src/app/services/alert/alert.service.ts

```diff
@@ -19,9 +19,10 @@ export class AlertService {
 
   public error(message: string): void {
     this._snackBar.openFromComponent(ErrorComponent, {
-      duration: 4000,
+      duration: 0,
       data: {
-        html: message
+        html: message,
+        duration: 4000
```

Comment:
nebude se to být, když se 2 věci jmenují `duration`? K cemu je to druhe, co je mimo `data`